### DISCUSSION TITLE
Serveur Cloudlink

### DISCUSSION THREAD

**Discussion by Eaielectronic** [community/community#129422] [Programming Help]
### Body

Bonjour, j'aimerais savoir comment envoyer un message privé si un client envoie la commande foo. J'ai utilisé cet exemple : https://github.com/MikeDev101/cloudlink/tree/master/python 

### Guidelines

- [x] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Eaielectronic** [comment]
Hello, I would like to know how to send a private message if a client sends the foo command. I used this example: https://github.com/MikeDev101/cloudlink/tree/master/python

**Comment by HemalathaT2** [comment]
To send a private message if a client sends the foo command, you can modify the foobar function in the example plugin to send a private message to the client 
class myplugin:
    def __init__(self, server, protocol):
        @server.on_command(cmd="foo", schema=protocol.schema)
        async def foobar(client, message):
            # Send a private message to the client
            await client.send({"cmd": "private_message", "message": "You sent the foo command!"})
